src/portable.rs

```diff
@@ -120,7 +119,7 @@ pub fn compress_xof(
     crate::platform::le_bytes_from_words_64(&state)
 }
 
-pub fn hash1<const N: usize>(
+const fn hash1<const N: usize>(
```

Comment:
Removal of `pub` intended?

Reply:
Not really, but since there are a few other backends with the same function and in all cases it is internal implementation without `pub` (here too), I decided to remove it for consistency too. Happy to restore if maintainers have strong opinion on this one.